source/_components/tts.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "Text-to-Speech (TTS)"
+description: "Instructions how to setup Text-to-Speech (TTS) with Home Assistant."
+date: 2016-12-13 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: 0.35
+---
+
+Text-to-speech (TTS) enables Home Assistant to speak to you.
+
+# {% linkable_title Cache %}
+
+The component have 2 cache. A long time cache on file system, they can controll with `cache` option on platform config or service call `say`. And a in memory cache for fast response to media player, they will be auto cleanup after several time.
+
+# {% linkable_title Configuring a `tts` platform %}
+
+To get started add the following lines to your `configuration.yaml` (example for google):
+
+```yaml
+# Example configuration.yaml entry for google tts service
+tts:
+  - platform: google
+```
+
+The following optional parameters can be used with any platform. However tts component will only look for global settings under the configuration of the first configured platform:
```

Comment:
Note about the implementation, not docs: Why are we adding this limitation to a new platform? It's something I understand is supposed to be fixed in other platforms.